### PR TITLE
added spinner and containers and some minor changes

### PR DESCRIPTION
**Pull request by Fayyaziii332**
Added Spinners to  signin and signup pages .
created containers for appjs and privateroute components
Did some changes for imports , properties etc.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 node_modules
 /client/node_modules
-/client/public
\ No newline at end of file
+/client/public
```
Comment (Komal-Tehreem) [inline .gitignore]: remove extra space and add empty line at the end of file.

File: client/src/App.js [hunk 2]
```diff
@@ -1,30 +1,13 @@
...
-  const decoded = jwt_decode(token);
-  store.dispatch(setCurrentUser(decoded));
-  const currentTime = Date.now() / 1000; // milliseconds
-  if (decoded.exp < currentTime) {
-    store.dispatch(logoutUser());
-    window.location.href = "./login";
-  }
-}
-
 class App extends React.Component {
```
Comment (Komal-Tehreem) [inline client/src/App.js]: remove extra line. There should be only one empty line at the end of file.

File: client/src/components/Signup/signup.js [hunk 3]
```diff
@@ -43,18 +43,19 @@ handleChange = e => {
   }
 
 onSubmit = e => {
-    // e.preventDefault();
+  console.log(this.props.loading)
```
Comment (Komal-Tehreem) [inline client/src/components/Signup/signup.js]: remove console.log. It should give lint error

File: client/src/components/Signup/signup.js [hunk 4]
```diff
@@ -67,11 +68,12 @@ return (
             </Link>
             </div>
             <div>
-              <img src="logo.png" style={{width:"45%"}} />
+              <img alt="logo" src="logo.png" style={{width:"45%"}} />
```
Comment (Komal-Tehreem) [inline client/src/components/Signup/signup.js]: avoid inline styling.

**Comment by Komal-Tehreem** [comment]
I think you are still working on organising directory structure. Let's have discussion related to few actioTypes like LOADING and adding http with each api call.

**Comment by Fayyaziii332** [comment]
Ok sure.

**Pull request merged**